deploy/project_config.yaml.schema

```diff
@@ -250,6 +250,45 @@ definitions:
             minimum: 10
             maximum: 600
 
+      dataproc:
+        type: array
+        description: List of dataproc clusters to create.
+        items:
+          type: object
+          additionalProperties: true
```

Comment:
Does this block need additionalProperties? If not, it is useful to set it to false to catch typos in configs.